vkcs/dataplatform/resource_cluster.go

```diff
@@ -391,6 +400,76 @@ func clusterUpdateConfigsUsers(ctx context.Context, client *gophercloud.ServiceC
 	return nil
 }
 
+func clusterUpdatePodGroups(ctx context.Context, client *gophercloud.ServiceClient, id string, dataPodGroupsRaw basetypes.ListValue, planPodGroupsRaw basetypes.ListValue) diag.Diagnostics {
+	var diags diag.Diagnostics
+	var d diag.Diagnostics
+
+	dataPodGroups, d := resource_cluster.ReadClusterPodGroups(ctx, dataPodGroupsRaw)
```

Comment:
could we add this to get cluster state to be able to auto fix scaling if it has been chaned outside of tf?